### ISSUE TITLE
Unquoted urls throw an unrelated error

### ISSUE THREAD

**Issue by mfields** [CSSLint/csslint#94]
The following lines were flagged as a "Parsing Error":

```
#test {
    background:url( controls.png );
}
```

I do not believe that there is an error present here.

If you quote the url, the error disappears.

```
#test {
    background:url( 'controls.png' );
}
```

As far as I know it is not necessary to quote urls passed to url().
